jina/executors/rankers/__init__.py

```diff
@@ -32,7 +32,9 @@ class Chunk2DocRanker(BaseRanker):
 
     required_keys = {'text'}  #: a set of ``str``, key-values to extracted from the chunk-level protobuf message
     COL_MATCH_PARENT_HASH = 'match_parent_hash'
+    COL_MATCH_PARENT_ID = 'match_parent_hash'
```

Comment:
In case someone wonders, what this is about: This enables us to first change everything in the hub from `COL_MATCH_PARENT_HASH` to `COL_MATCH_PARENT_ID` without breaking and then remove the `COL_MATCH_PARENT_ID`, since it is an `id` again.